src/mmw/apps/user/models.py

```diff
@@ -72,3 +75,23 @@ class UserProfile(models.Model):
     country = models.TextField(choices=countries.COUNTRY_CHOICES,
                                default=countries.US)
     postal_code = models.TextField(blank=True)
+
+
+class HydroShareToken(models.Model):
+    user = models.OneToOneField(User, primary_key=True)
+    access_token = models.CharField(max_length=255)
+    token_type = models.CharField(max_length=255, default="Bearer")
+    expires_in = models.IntegerField(default=0)
+    refresh_token = models.CharField(max_length=255, null=True)
```

Comment:
Null char fields are discouraged by Django because it introduces ambiguity between `''` and `null` values.

Will there be a situation where `refresh_token` be is null? Based on testing, every time a row is written or updated in `user_hydrosharetoken` it always has a `refresh_token` value.

Reply:
The [HS RestClient docs](http://hs-restclient.readthedocs.io/en/latest/) don't talk of a refresh token anywhere, so even though it is available in the API it felt less official than the `access_token`. But I agree, given that the API itself has it, this is probably just lacking in the RestClient.

I'll make it not NULL.